### ISSUE TITLE
Investigate native video player screen reader compatibility and raise bug reports

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-design-system#911]
## What

We've had reports that the native video players we're using on the design system website are not accessible to screen reader users: https://github.com/alphagov/govuk-design-system/issues/904

## Why

Make want to sure that browser vendors are aware of the issues so they can be resolved.

## Done when

- [ ] Check our support matrix for issues in native video player
- [ ] If there are issues raise an issue with the browser vendor (https://github.com/alphagov/reported-bugs)

**Comment by nickcolley** [comment]
This is blocked on JAWs 2019 since Edge does not work in older versions of JAWs.

We'd also need help from the accessibility team to get admin permissions to install Edge Chromium.

I'm going to move this out of the sprint until we've got the latest JAWs.

**Comment by nickcolley** [comment]
I've expanded the scope of this card to be a general review from the recommendation of Anika

**Comment by nickcolley** [comment]
This blog post details many of the issues with the native video player and then recommends a third party which has it's own issues: https://scottvinkle.me/blogs/blog/how-accessible-is-the-html-video-player

**Comment by timpaul** [comment]
Investigation completed. We have decided not to raise bug reports as the issues are already well documented elsewhere.
